packages/m/miaoyan.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "miaoyan",
+  "description": "Lightweight Markdown app to help you write great sentences.",
+  "filename": "index.min.js",
```

Comment:
This filename should match a file in your project, the default file that folks should be importing into a browser to start using your project.